juju/juju
# Generations Command "cancel-generation" Becomes "commit"

## Description of change

Depends on #9972.

This patch renames the `cancel-generation` command to `commit`, including:
- File-name changes.
- Help text and command description changes.
- API method renaming.
- Modified comments.

## QA steps

- `export JUJU_DEV_FEATURE_FLAGS=generations`
- Bootstrap.
- `juju branch test-branch`
- `juju commit test-branch` should indicate that the branch was aborted.
- `juju deploy redis`
- `juju branch test-branch`
- `config redis password=pass --branch test-branch`
- `juju commit test-branch` should indicate that the branch was committed and show the generation ID.

## Documentation changes

None

## Bug reference

N/A
